extensions/amp-list/0.1/amp-list.js

```diff
@@ -72,6 +73,10 @@ export class AmpList extends AMP.BaseElement {
 
     /** @const @private {string} */
     this.initialSrc_ = element.getAttribute('src');
+
+    this.registerAction('refresh', () => {
+      this.fetchList_();
```

Comment:
we should wrap this in `if(this.layoutCompleted_){}` I don't think this action should be allowed to do anything before `amp-list` is laidout (e.g. be used for initial load). If we need to support that use-case, there is a bit more we need to do than just call `fetchList_`

Reply:
Makes sense. Done.